tools/UpdatePaths/Scripts/78757_simple_to_basic_heretic_summons.txt

```diff
@@ -0,0 +1,4 @@
+/mob/living/simple_animal/hostile/heretic_summon/ash_spirit : /mob/living/basic/heretic_summon/ash_spirit
+/mob/living/simple_animal/hostile/heretic_summon/maid_in_the_mirror : /mob/living/basic/heretic_summon/maid_in_the_mirror
+/mob/living/simple_animal/hostile/heretic_summon/rust_spirit : /mob/living/basic/heretic_summon/rust_walker{@OLD;AIStatus=@SKIP;stop_automated_movement=@SKIP}
+/mob/living/simple_animal/hostile/heretic_summon/stalker : /mob/living/basic/heretic_summon/stalker
```

Comment:
why not have {@OLD} on the right-hand-side of the other three path replacements here?

Reply:
Because I don't respect anyone who put them into a map